Delivered-To: mine260309@gmail.com
Received: by 2002:a4f:7f0c:0:0:0:0:0 with SMTP id a12csp59678ivd;
        Thu, 3 Jan 2019 13:55:20 -0800 (PST)
X-Google-Smtp-Source: AFSGD/VYxBqyK1BgGjkV+OcctTlP5fIa4Xhx8cHXpvqhfHEEEqRQsMKwQ5E8EVmiJ3eX6pOqtXd0
X-Received: by 2002:a62:8949:: with SMTP id v70mr48781241pfd.85.1546552520656;
        Thu, 03 Jan 2019 13:55:20 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1546552520; cv=none;
        d=google.com; s=arc-20160816;
        b=LsGeQ2/iF27QmCv52gWLd0yBkPihHD5ie9mvUUd39jxHVxVUkHgDFlAXKvhju38Br1
         PGwX7kKF5pI/QFUk3ggiF9Xia2wac98zwflsHR+uFUhK0zWSBEenXOKt7/nd1we1+Ocr
         1eQiwJRXzDxrtUW/lF/TmduNSmVZVIOmaCJiJaeOUa4MW5dqLMu4geKOdZi0OU50TZjm
         KTFR9sXLalrnuPdxCr1E/fM5cxahRJgj+1ukw8bMvceaQAcZdXK2ITHxRB8kS+cF66aE
         CNvLKOmhrgJRaVvioYb+5vvmTtDfzSS7XVpkKj9Yw6KEVXjMRS+A7xGTCGlqdzj3MZ6w
         pDTw==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=sender:errors-to:cc:list-subscribe:list-help:list-post:list-archive
         :list-unsubscribe:list-id:precedence:message-id:references
         :in-reply-to:date:subject:to:from:delivered-to;
        bh=8HdO89C8/YlPRV56coUsh9d9yF0Kd7RZudedpRgyYDo=;
        b=yJJ9oP3siuSOndtFCX+OTUco2vV7UmfMv2xzvufYhRh0RoDNTOHSkZRyjC+1luSK66
         Ku3oiYhAjo4nh3rXMIR8oHtu9+6quR+uYlj8PD9oIqmUIF+VcFvBEthmjr7hZGcNAlKd
         1icovOPnWezRHfjw2UHw4YaZFkzxSL+pZUkVvWWwakeGSGkrB0UIFoKvd1GR2AFCZl0D
         pwGSSVjKNHxphXiC81Ik97+xrOlcrdYnxB2X5vPFH5vx9P6lSQx8LkM20GD0Amkl5tPX
         xZUPKixhd/4jUqNx0FXvLcHILBJS1Fm6yGVguKDVvtZ0XBG4+e6BM9dQ/FRuo5kmQb7Y
         R5xQ==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Return-Path: <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>
Received: from lists.ozlabs.org (lists.ozlabs.org. [203.11.71.2])
        by mx.google.com with ESMTPS id v19si55973448pfa.80.2019.01.03.13.55.20
        for <mine260309@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-CHACHA20-POLY1305 bits=256/256);
        Thu, 03 Jan 2019 13:55:20 -0800 (PST)
Received-SPF: pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) client-ip=203.11.71.2;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Received: from lists.ozlabs.org (lists.ozlabs.org [IPv6:2401:3900:2:1::3])
	by lists.ozlabs.org (Postfix) with ESMTP id 43W1tY6c3bzDqLY
	for <mine260309@gmail.com>; Fri,  4 Jan 2019 08:55:17 +1100 (AEDT)
X-Original-To: openbmc@lists.ozlabs.org
Delivered-To: openbmc@lists.ozlabs.org
Authentication-Results: lists.ozlabs.org;
 spf=pass (mailfrom) smtp.mailfrom=linux.ibm.com
 (client-ip=148.163.158.5; helo=mx0a-001b2d01.pphosted.com;
 envelope-from=eajames@linux.ibm.com; receiver=<UNKNOWN>)
Authentication-Results: lists.ozlabs.org;
 dmarc=none (p=none dis=none) header.from=linux.ibm.com
Received: from mx0a-001b2d01.pphosted.com (mx0b-001b2d01.pphosted.com
 [148.163.158.5])
 (using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
 (No client certificate requested)
 by lists.ozlabs.org (Postfix) with ESMTPS id 43W1sV18w7zDqDF
 for <openbmc@lists.ozlabs.org>; Fri,  4 Jan 2019 08:54:21 +1100 (AEDT)
Received: from pps.filterd (m0098421.ppops.net [127.0.0.1])
 by mx0a-001b2d01.pphosted.com (8.16.0.22/8.16.0.22) with SMTP id
 x03LsI0p047720
 for <openbmc@lists.ozlabs.org>; Thu, 3 Jan 2019 16:54:19 -0500
Received: from e32.co.us.ibm.com (e32.co.us.ibm.com [32.97.110.150])
 by mx0a-001b2d01.pphosted.com with ESMTP id 2pspsu1jbp-1
 (version=TLSv1.2 cipher=AES256-GCM-SHA384 bits=256 verify=NOT)
 for <openbmc@lists.ozlabs.org>; Thu, 03 Jan 2019 16:54:19 -0500
Received: from localhost
 by e32.co.us.ibm.com with IBM ESMTP SMTP Gateway: Authorized Use Only!
 Violators will be prosecuted
 for <openbmc@lists.ozlabs.org> from <eajames@linux.ibm.com>;
 Thu, 3 Jan 2019 21:54:18 -0000
Received: from b03cxnp08028.gho.boulder.ibm.com (9.17.130.20)
 by e32.co.us.ibm.com (192.168.1.132) with IBM ESMTP SMTP Gateway: Authorized
 Use Only! Violators will be prosecuted; 
 (version=TLSv1/SSLv3 cipher=AES256-GCM-SHA384 bits=256/256)
 Thu, 3 Jan 2019 21:54:17 -0000
Received: from b03ledav003.gho.boulder.ibm.com
 (b03ledav003.gho.boulder.ibm.com [9.17.130.234])
 by b03cxnp08028.gho.boulder.ibm.com (8.14.9/8.14.9/NCO v10.0) with ESMTP id
 x03LsGYT20906144
 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-GCM-SHA384 bits=256 verify=FAIL);
 Thu, 3 Jan 2019 21:54:16 GMT
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 4F9A56A054;
 Thu,  3 Jan 2019 21:54:16 +0000 (GMT)
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 0D1B36A04F;
 Thu,  3 Jan 2019 21:54:16 +0000 (GMT)
Received: from talon7.ibm.com (unknown [9.41.179.222])
 by b03ledav003.gho.boulder.ibm.com (Postfix) with ESMTP;
 Thu,  3 Jan 2019 21:54:15 +0000 (GMT)
From: Eddie James <eajames@linux.ibm.com>
To: openbmc@lists.ozlabs.org
Subject: [PATCH dev-4.19 3/6] ARM: dts: aspeed-g5: Add video engine node
Date: Thu,  3 Jan 2019 15:54:05 -0600
X-Mailer: git-send-email 1.8.3.1
In-Reply-To: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
References: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
X-TM-AS-GCONF: 00
x-cbid: 19010321-0004-0000-0000-000014CC6DA7
X-IBM-SpamModules-Scores: 
X-IBM-SpamModules-Versions: BY=3.00010339; HX=3.00000242; KW=3.00000007;
 PH=3.00000004; SC=3.00000272; SDB=6.01141311; UDB=6.00593987; IPR=6.00921536; 
 MB=3.00024957; MTD=3.00000008; XFM=3.00000015; UTC=2019-01-03 21:54:18
X-IBM-AV-DETECTION: SAVI=unused REMOTE=unused XFE=unused
x-cbparentid: 19010321-0005-0000-0000-00008A14BAD8
Message-Id: <1546552448-22621-4-git-send-email-eajames@linux.ibm.com>
X-Proofpoint-Virus-Version: vendor=fsecure engine=2.50.10434:, ,
 definitions=2019-01-03_10:, , signatures=0
X-Proofpoint-Spam-Details: rule=outbound_notspam policy=outbound score=0
 priorityscore=1501
 malwarescore=0 suspectscore=1 phishscore=0 bulkscore=0 spamscore=0
 clxscore=1015 lowpriorityscore=0 mlxscore=0 impostorscore=0
 mlxlogscore=999 adultscore=0 classifier=spam adjust=0 reason=mlx
 scancount=1 engine=8.0.1-1810050000 definitions=main-1901030187
X-BeenThere: openbmc@lists.ozlabs.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: Development list for OpenBMC <openbmc.lists.ozlabs.org>
List-Unsubscribe: <https://lists.ozlabs.org/options/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=unsubscribe>
List-Archive: <http://lists.ozlabs.org/pipermail/openbmc/>
List-Post: <mailto:openbmc@lists.ozlabs.org>
List-Help: <mailto:openbmc-request@lists.ozlabs.org?subject=help>
List-Subscribe: <https://lists.ozlabs.org/listinfo/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=subscribe>
Cc: Eddie James <eajames@linux.vnet.ibm.com>
Errors-To: openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org
Sender: "openbmc" <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>

From: Eddie James <eajames@linux.vnet.ibm.com>

Add a node to describe the video engine on the AST2500.

Signed-off-by: Eddie James <eajames@linux.vnet.ibm.com>
---
 arch/arm/boot/dts/aspeed-g5.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-g5.dtsi b/arch/arm/boot/dts/aspeed-g5.dtsi
index f6e630f..8000877 100644
--- a/arch/arm/boot/dts/aspeed-g5.dtsi
+++ b/arch/arm/boot/dts/aspeed-g5.dtsi
@@ -243,6 +243,17 @@
 				interrupts = <0x19>;
 			};
 
+			video: video@1e700000 {
+				compatible = "aspeed,ast2500-video-engine";
+				reg = <0x1e700000 0x20000>;
+				clocks = <&syscon ASPEED_CLK_GATE_VCLK>,
+					 <&syscon ASPEED_CLK_GATE_ECLK>;
+				clock-names = "vclk", "eclk";
+				resets = <&syscon ASPEED_RESET_VIDEO>;
+				interrupts = <7>;
+				status = "disabled";
+			};
+
 			adc: adc@1e6e9000 {
 				compatible = "aspeed,ast2500-adc";
 				reg = <0x1e6e9000 0xb0>;
-- 
1.8.3.1

